docs/docsource/Installation/upgrade/upgrading_zenko.rst

```diff
@@ -16,87 +16,90 @@ To upgrade Zenko:
 
       $ helm upgrade {{zenko-server-name}} ./zenko
 
-   If you are using a custom values be sure to reuse this file on upgrades.
+   If you are using custom values, reuse options.yml on upgrades.
    ::
 
-      $ helm upgrade {{zenko-server-name}} ./zenko -f options.yaml
+      $ helm upgrade {{zenko-server-name}} ./zenko -f options.yml
 
-   On success, Helm reports:
+#. On success, Helm reports:
    :: 
 
       Release "{{zenko-server-name}}" has been upgraded. Happy Helming!
 
-   Then, after a few seconds, Helm displays a voluminous status report on the
+   After a few seconds, Helm displays a voluminous status report on the
    server's current operating condition.
 
-   ..tip ::
+   .. tip::
 
       Expect the cluster to take a few minutes to stabilize. You may see 
       CrashLoopBackoff errors. This is normal, expected behavior.
 
 Upgrading from 1.0.x to 1.1.x
-=============================
+-----------------------------
+
+Zenko 1.0.x versions use MongoDB version 3.4, which has been upgraded to 3.6
+in Zenko 1.1.x. Although upgrades using the commands above will work, some
+newer features may not function.
 
-Zenko 1.0.x versions use MongoDB version 3.4 which is upgraded to 3.6 in Zenko
-1.1.x versions. Upgrades using the previously mentioned commands will work
-however some newer features may not function by default.
+To upgrade from 1.0.x to 1.1:
 
 #. Run the typical upgrade command, inserting your Zenko server's release name
    ::
 
      $ helm upgrade {{zenko-release-name}} ./zenko
 
-   If you are using a custom values be sure to reuse this file on upgrades.
+   If you are using custom values, be sure to reuse the options.yaml file on
+   upgrades.
    ::
 
       $ helm upgrade {{zenko-server-name}} ./zenko -f options.yaml
 
-   After the upgrade has stabilized all pod rollouts, run the following
-   command to finialize the MongoDB compatibility upgrade:
+#. After the upgrade has stabilized all pod rollouts, run the following
+   command to finalize the MongoDB compatibility upgrade:
    ::
 
      $ helm upgrade {{zenko-release-name}} ./zenko --set maintenance.upgradeMongo=true -f options.yaml
 
-   You will see a pod get deployed that will run to and show a Completed status
-   once the upgrade has successfully occurred.
+#. A pod is deployed. When the upgrade is successful, it shows a "Completed"
+   status.
    ::
 
      {{zenko-release-name}}-zenko-mongo-capabilities       0/1     Completed          0          4h
 
-   ..note ::
+   .. note::
 
-     Once the upgrade is completed the maintenance flag is not needed
-     and should not be specified for further 1.1.x upgrades.
+      Once this upgrade is complete, the maintenance flag is no
+      longer required for further 1.1.x upgrades.
 
-Upgrade conisderations
-======================
+Upgrade Considerations
```

Comment:
Shouldn't upgrade considerations and dry-run come before the actual upgrade command.
If the user is following the manual step by step, then they might miss this part.